README.md

```diff
@@ -45,11 +45,25 @@ Once your docker is running, you can configure the Wordpress installation:
 For unit testing a separate CLI based wordpress is included, this can be started trough
 
 ```CLI
-$docker compose -f docker-compose-cli.yml up
+$ docker compose -f docker-compose-cli.yml up
 ```
 
 You can than run [Wordpress CLI commands](https://developer.wordpress.org/cli/commands/) with in the container. For example, getting a list of all available plugins can be done trough
```

Comment:
You can (then) run [Wordpress CLI commands](https://developer.wordpress.org/cli/commands/) within the container. For example, getting a list of all available plugins can be done through.